modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/styles/components/_sla-card.scss

```diff
@@ -0,0 +1,48 @@
+.sla-container:hover {
+	.btn {
+		background-color: white;
+		border-color: rgba(11, 95, 255, 0.25);
+		box-shadow: 0 10px 20px 0 #0b5fff1c;
+		display: inline-block !important;
+		padding: 0.25rem;
+		transform: translate(12rem, -3rem);
+		z-index: 100;
+	}
+	.btn:hover {
+		border-color: white;
+	}
+}
+.card {
+	box-shadow: 0 10px 20px 0 #0b5fff12;
```

Comment:
Don't we have this in Dialect box-shadows? Take a look in common/styles/_variables.scss, we have a very similar box shadow in a variable $elevation-fluffy-3. You can add a new one like $elevation-fluffy-2 and use here

Reply:
Thanks @iagochaves !